volume/volume.go

```diff
@@ -73,10 +72,8 @@ func (m *MountPoint) Setup() (string, error) {
 				return "", err
 			}
 			if runtime.GOOS != "windows" { // Windows does not have deprecation issues here
```

Comment:
I think you can remove the runtime check now. This should apply to both Windows and Linux (I believe @cpuguy83)